Name: Reply2All for Move v2
Version: 1.3.5-1
Author: Jason Robitaille, dBsooner ,mikepre
Description: This will replace the 'Reply to All' button with the 'Move to Folder' menu option. The 'Reply to All' is moved to the menu. Add shortcut to ReplyAll, change shortcut for SetFlag to 'f', change shortcut for mark unread to 'u'. Cannot have both Reply2All patches applied at same time

--- .orig/usr/palm/applications/com.palm.app.email/app/controllers/message-assistant.js
+++ /usr/palm/applications/com.palm.app.email/app/controllers/message-assistant.js
@@ -72,6 +72,7 @@
 		this.cmdMenuModel = {
 				visible:true,
 				items: [
+					{label:$L('Move'),      icon:'file',      command:'move'},
 					{label:$L('Reply'),     icon:'reply',     command:'reply'},
 					{label:$L('Reply all'), icon:'reply-all', command:'replyAll'},
 					{label:$L('Forward'),   icon:'forward-email', command:'forward'},
@@ -73,7 +74,6 @@
 				visible:true,
 				items: [
 					{label:$L('Reply'),     icon:'reply',     command:'reply'},
-					{label:$L('Reply all'), icon:'reply-all', command:'replyAll'},
 					{label:$L('Forward'),   icon:'forward-email', command:'forward'},
 					{label:$L('Delete'),    icon:'delete',    command:'delete'}
 				]};
@@ -79,8 +79,8 @@
 				]};
 		this.controller.setupWidget(Mojo.Menu.commandMenu, undefined, this.cmdMenuModel);
 		
-		this.markUnreadMenuItem =     {label:MessageAssistant.kAppMenuMarkUnread, shortcut:'k', command:'mark-unread'};
-		this.markSetFlagMenuItem =    {label:MessageAssistant.kAppMenuSetFlag, shortcut:'g', command:'flag'};
+		this.markUnreadMenuItem =     {label:MessageAssistant.kAppMenuMarkUnread, shortcut:'u', command:'mark-unread'};
+		this.markSetFlagMenuItem =    {label:MessageAssistant.kAppMenuSetFlag, shortcut:'f', command:'flag'};
 		this.showRecipientsMenuItem = {label:MessageAssistant.kAppMenuShowRecipients, shortcut:'s', command:'show-recipients'};
 		
 		this.appMenuAttrs = {
@@ -106,7 +106,7 @@
 				emailEditItems,
 				this.markUnreadMenuItem,
 				this.markSetFlagMenuItem,
-				{label:$L('Move to Folder...'), shortcut:'m', command:'move'},
+				{label:$L('Reply all'), shortcut:'r',command:'replyAll'},
 				this.showRecipientsMenuItem,
 				Mojo.Menu.prefsItem,
 				Mojo.Menu.helpItem
